README.md

```diff
@@ -191,6 +192,49 @@ By using this tool you accept this risk and you take full responsibility for the
     ```
     where `<reg_address>` is register address (decimal), and <reg_value> is a value to set (decimal)
 
+## Using the stdout publisher
```

Comment:
This file is already too large. Would you mind to shorten this section and move the details into a new file under `docs` folder?

Reply:
Spit the stdout publisher docs into its own file, doc'ing the telegraf example there too.